pallets/gear/src/lib.rs

```diff
@@ -562,7 +562,29 @@ pub mod pallet {
                     )
                 };
 
-                core_processor::handle_journal(journal.clone(), &mut ext_manager);
+                // TODO: Check whether we charge gas fee for submitting code after #646
+                let notes = journal
+                    .into_iter()
+                    .filter_map(|note| -> Option<Result<JournalNote, _>> {
+                        match note {
+                            JournalNote::GasBurned { .. }
+                            | JournalNote::WakeMessage { .. }
+                            | JournalNote::SendDispatch { .. }
+                            | JournalNote::WaitDispatch(..) => Some(Ok(note)),
```

Comment:
please review these, removed `MessageConsumed` because it doesn't spend any gas on current node